cmd/revision.go

```diff
@@ -0,0 +1,112 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/databus23/helm-diff/diff"
+	"github.com/databus23/helm-diff/manifest"
+	"github.com/spf13/cobra"
+	"k8s.io/helm/pkg/helm"
+)
+
+type revision struct {
+	release         string
+	client          helm.Interface
+	suppressedKinds []string
+	revisions       []string
+}
+
+const revisionCmdLongUsage = `
+This command compares the manifests details of a named release.
+
+It can be used to compare the manifests of 
+ 
+ - lastest REVISION with specified REVISION
+	$ helm diff revision [flags] [RELEASE] [REVISION]
+
+ - REVISION1 with REVISION2
+	$ helm diff revision [flags] [RELEASE] [REVISION1] [REVISION2]
+`
+
+func revisionCmd() *cobra.Command {
+	diff := revision{}
+	revisionCmd := &cobra.Command{
+		Use:   "revision [flags] [RELEASE] REVISION1 REVISION2",
```

Comment:
When doing some manually tests I was wondering if `revision [flags] RELEASE REVISION1 [REVISION2]` would be more correct?

Reply:
both revision1 and revision2 are mandatory, so how can it be indicated..?

Reply:
It seems to me like REVISION 2 is not mandatory and defaults to the latest revision. At least I can do `helm diff revision $RELEASE 42`